### PR TITLE
replace Slack links with discord links

### PR DESCRIPTION
**Pull request by janmyszkier**
### Related Issues
None

closes #

### Short Description of the PR
Replace support channel info with Discord

### Pull Request Checklist

- [ ] I have updated the Changelog ([V1](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md)) [v2](https://docs-next.vuestorefront.io/contributing/creating-changelog.html) and mentioned all breaking changes in the public API.
- [ ] I have documented all new public APIs and made changes to existing docs mentioning the parts I've changed so they're up to date.
- [ ] I have tested my Pull Request on production build and (to my knowledge) it works without any issues

- [ ] I have followed [naming conventions](https://github.com/kettanaito/naming-cheatsheet)

### CODE REVIEW

File: packages/core/docs/.vuepress/components/PersonTile.vue [hunk 1]
```diff
@@ -3,13 +3,13 @@
     <img :src="photo" />
     <div><b> {{ name }} </b></div>
     <div class="gap">{{ company }} </div>
-    <div><b><a href="https://slack.vuestorefront.io">Slack:</a></b> @{{ slack }} </div>
+    <div><b><a href="https://discord.vuestorefront.io">Discord:</a></b> @{{ discord }} </div>
```
Comment (filipsobol) [inline packages/core/docs/.vuepress/components/PersonTile.vue:6]: Please remove `@`

File: packages/core/docs/enterprise/feature-list.md [hunk 2]
```diff
@@ -349,7 +349,7 @@ All Vue Storefront Cloud config options are available via API
 
 **14. SLA & Support**
 
-The Storefront Cloud team will guarantee 99,8% SLA for the infrastructure and Vue Storefront base app (modifications and custom integrations created by merchant/agency are excluded). Our clients can use our dedicated Slack support channel to contact the team and check the tickets’ statuses.
+The Storefront Cloud team will guarantee 99,8% SLA for the infrastructure and Vue Storefront base app (modifications and custom integrations created by merchant/agency are excluded). Our clients can use our dedicated Discord support channel to contact the team and check the tickets’ statuses.
```
Comment (filipsobol) [inline packages/core/docs/enterprise/feature-list.md:352]: Our clients have dedicated Slack channels which we use internally. Please revert this change.

File: packages/core/docs/.vuepress/components/PersonTile.vue (hunk 1 above)
Comment (filipsobol) [inline packages/core/docs/.vuepress/components/PersonTile.vue:6]:
```suggestion
    <div><b><a href="https://discord.vuestorefront.io">Discord:</a></b> {{ discord }} </div>
```

File: packages/core/docs/commercetools/maintainers.md [hunk 3]
```diff
@@ -5,11 +5,11 @@ The Commercetools integration is developed and maintained by **Vue Storefront co
...
+In case of any questions don't hesitate reaching out to the maintainers on [Discord](https://discord.vuestorefront.io) to support you in delivering your project.
 
 <PersonTile 
   photo="https://avatars3.githubusercontent.com/u/7943292?s=460&u=74f1c2a48fdbac23962aa3b1f7346bc0c9e55309&v=4"
   name="Patryk Andrzejewski"
   company="Vue Storefront"
-  slack="andrzejewsky"
-/>
\ No newline at end of file
+  discord="andrzejewsky"
```
Comment (filipsobol) [inline packages/core/docs/commercetools/maintainers.md:14]:
```suggestion
  discord="andrzejewsky#9834"
```

File: packages/core/docs/enterprise/feature-list.md (hunk 2 above)
Comment (filipsobol) [inline packages/core/docs/enterprise/feature-list.md:352]:
```suggestion
The Storefront Cloud team will guarantee 99,8% SLA for the infrastructure and Vue Storefront base app (modifications and custom integrations created by merchant/agency are excluded). Our clients can use our dedicated Slack support channel to contact the team and check the tickets’ statuses.
```

File: packages/core/docs/integrate/cache-driver.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # Integrating cache driver
 
 ::: warning Want to build an integration?
-If you want to integrate with Vue Storefront, please **contact the core team on our [slack](https://slack.vuestorefront.io)** first. We are eager to help you with building it and ensuring its high quality! Building the integration together with the core team is the best way to keep its quality high and make it officially recommended once it's done.
+If you want to integrate with Vue Storefront, please **contact the core team on our [discord](https://discord.vuestorefront.io)** first. We are eager to help you with building it and ensuring its high quality! Building the integration together with the core team is the best way to keep its quality high and make it officially recommended once it's done.
```
Comment (filipsobol) [inline packages/core/docs/integrate/cache-driver.md:4]:
```suggestion
If you want to integrate with Vue Storefront, please **contact the core team on our [Discord](https://discord.vuestorefront.io)** first. We are eager to help you with building it and ensuring its high quality! Building the integration together with the core team is the best way to keep its quality high and make it officially recommended once it's done.
```

File: packages/core/docs/integrate/cms.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 # Integrating CMS
 
 ::: warning Want to build an integration?
-If you want to integrate with Vue Storefront, please **contact the core team on our [slack](https://slack.vuestorefront.io)** first. We are eager to help you with building it and ensuring its high quality! Building the integration together with the core team is the best way to keep its quality high and make it officially recommended once it's done.
+If you want to integrate with Vue Storefront, please **contact the core team on our [discord](https://discord.vuestorefront.io)** first. We are eager to help you with building it and ensuring its high quality! Building the integration together with the core team is the best way to keep its quality high and make it officially recommended once it's done.
```
Comment (filipsobol) [inline packages/core/docs/integrate/cms.md:4]:
```suggestion
If you want to integrate with Vue Storefront, please **contact the core team on our [Discord](https://discord.vuestorefront.io)** first. We are eager to help you with building it and ensuring its high quality! Building the integration together with the core team is the best way to keep its quality high and make it officially recommended once it's done.
```

File: packages/core/docs/integrate/integration-guide.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 # Integration guide
 
 ::: warning Want to build an integration?
-If you want to integrate with Vue Storefront, please **contact the core team on our [slack](https://slack.vuestorefront.io)** first. We are eager to help you with building it and ensuring its high quality! Building the integration together with the core team is the best way to keep its quality high and make it officially recommended once it's done.
+If you want to integrate with Vue Storefront, please **contact the core team on our [discord](https://discord.vuestorefront.io)** first. We are eager to help you with building it and ensuring its high quality! Building the integration together with the core team is the best way to keep its quality high and make it officially recommended once it's done.
```
Comment (filipsobol) [inline packages/core/docs/integrate/integration-guide.md:4]:
```suggestion
If you want to integrate with Vue Storefront, please **contact the core team on our [Discord](https://discord.vuestorefront.io)** first. We are eager to help you with building it and ensuring its high quality! Building the integration together with the core team is the best way to keep its quality high and make it officially recommended once it's done.
```

File: packages/core/docs/shopify/maintainers.md [hunk 7]
```diff
@@ -6,11 +6,11 @@ The Shopify integration is developed and maintained by the core partner **Aureat
...
+In case of any questions don't hesitate reaching out to the maintainers on [Discord](https://discord.vuestorefront.io) and visiting `#shopify` channel. You can also [hire the core team](https://www.vuestorefront.io/support) to support you in delivering your project.
 
 <PersonTile 
   photo="https://avatars2.githubusercontent.com/u/783102?s=460&u=38994305c6b6ce2be5519544251d6875263dfb1a&v=4"
   name="Yogesh Suhagiya"
   company="Aureate Labs"
-  slack="yogeshsuhagiya"
-/>
\ No newline at end of file
+  discord="yogeshsuhagiya"
```
Comment (filipsobol) [inline packages/core/docs/shopify/maintainers.md:15]:
```suggestion
  discord="viralr07#1995"
```

**Review by filipsobol — CHANGES_REQUESTED** [review]

**Comment by filipsobol** [comment]
Thank you for this contribution. I added few suggestions

**Comment by filrak** [comment]
Wow, that's a lot of work, thanks so much @janmyszkier ❤️

**Comment by filipsobol** [comment]
Closing due to inactivity. Updating the links will be done in this PR https://github.com/vuestorefront/vue-storefront/pull/5888

**Pull request closed without merging**